### PR TITLE
Add __init__.py files to "frequenz/api/common/v1" folder

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: py/frequenz/api/common/v1/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# License: MIT
+# Copyright © 2023 Frequenz Energy-as-a-Service GmbH
```
Comment (nhatcher-frequenz) [inline py/frequenz/api/common/v1/__init__.py]: Wrong year?

also in all the others

**Review by nhatcher-frequenz — CHANGES_REQUESTED** [review]
Small issue

File: py/frequenz/api/common/v1/__init__.py (hunk 1 above)
Comment (ghost) [reply py/frequenz/api/common/v1/__init__.py]: uups good call, fixed now

**Review by nhatcher-frequenz — APPROVED** [review]
LGTM!

**Pull request merged**